source/javascripts/controllers/_WorkflowsController.js.erb

```diff
@@ -204,6 +205,7 @@ import { safeDigest } from "../services/react-compat";
 										}
 									);
 								} catch (error) {
+									loggerService.error(error.message, error);
```

Comment:
what does it log here? should we use `reject` part of the promise? Also couple lines above there is a `console.error` entry should we replace that as well? :)